hasura/seeds/default/1676722071560_teamSeed.sql

```diff
@@ -1,2 +1,3 @@
 SET check_function_bodies = false;
 INSERT INTO public.team (id, name, created_at, updated_at) VALUES ('team_d7cde14f17eda7e0ededba7ded6b4467', 'Testing Co', '2023-02-18 10:44:27.212563+00', '2023-02-18 10:44:27.212563+00');
+INSERT INTO public.team (id, name, created_at, updated_at) VALUES ('team_d7cde14f17eda7e0ededba7ded6b4468', 'Testing Co', '2024-01-01 10:44:27.212563+00', '2024-01-01 10:44:27.212563+00');
```

Comment:
```suggestion
INSERT INTO public.team (id, name, created_at, updated_at) VALUES ('team_2222214f17eda7e0ededba7ded6b4222', 'Testing Co #2', '2024-01-01 10:44:27.212563+00', '2024-01-01 10:44:27.212563+00');
```
minor thing to make it clearer